packages/nextjs/pages/index.tsx

```diff
@@ -209,37 +217,125 @@ const Home: NextPage<{ stats: Stats }> = ({ stats }) => {
 
       {/* Supporting Devs*/}
       <div className="bg-base-300">
-        <div className="mx-auto lg:max-w-[1980px] bg-none lg:bg-[url('/assets/funding.png')] md:[background-position-x:40vw] md:bg-contain bg-no-repeat xl:bg-right">
+        <div className="mx-auto lg:max-w-[1980px] bg-none lg:bg-[url('/assets/support-high-impact-devs.png')] md:[background-position-x:45vw] [background-position-y:50%] md:bg-auto bg-no-repeat ">
```

Comment:
Was wondering if we should keep this full image similar to other section, also found in Figma we have it similar to the other section (lol maybe Andrea was tweaking a few things while I was viewing) : 
![Screenshot 2023-09-18 at 8 50 10 PM](https://github.com/BuidlGuidl/buidlguidl.com/assets/80153681/a509b79a-4627-4b0e-85d1-0b0463c378ad)

But I think I like it how we have it for other sections. 

Also noticed that for some reason website image looks a bit blurry : 
![Screenshot 2023-09-18 at 8 57 40 PM](https://github.com/BuidlGuidl/buidlguidl.com/assets/80153681/fdaa03de-b750-49e3-8eec-b862ee9f1350)

Reply:
Just tried this section making similar to other sections at [07a8cde](https://github.com/BuidlGuidl/buidlguidl.com/pull/19/commits/07a8cde258223e71e982d7e3e895846b4e9c02c9) as discussed in TG  : 

![Screenshot 2023-09-20 at 7 41 40 PM](https://github.com/BuidlGuidl/buidlguidl.com/assets/80153681/ff220e27-33ef-44a2-9ebd-426ca35d3151)


https://buidlguidl-gmcjv9kop-buidlguidldao.vercel.app/

But on screens b/w `1024 - 1180` it looks a bit weird since we have alot description in this section as compared to other sections : 

| Current section | Other similar section with less description | 
| --------------- | ------------------------------------------ | 
|  <video src="https://github.com/BuidlGuidl/buidlguidl.com/assets/80153681/3811132a-b4ab-4a83-abdf-4ff0bfe5a2fe">  | <video src="https://github.com/BuidlGuidl/buidlguidl.com/assets/80153681/fe069463-544e-40d4-94de-46ccee12d986">   | 

I have commented on your code below, lol was not sure which looks better maybe we could ask Andrea for suggestions too?

Reply:
Thanks for playing with it and proposing a new version, I prefer your approach ♥. 

I'll ask Andrea too 🙌

Reply:
The old version doesn't display correctly in super big screens;

![image](https://github.com/BuidlGuidl/buidlguidl.com/assets/2486142/27a4832d-7204-4200-96c3-781f4a6d1c2e)

But it's fixed in the new one!